utils/src/merkle_tree/full_merkle_tree.rs

```diff
@@ -5,11 +5,10 @@ use std::{
     str::FromStr,
 };
 
-use color_eyre::{Report, Result};
 use rayon::iter::{IntoParallelIterator, ParallelIterator};
 
 use crate::merkle_tree::{FrOf, Hasher, ZerokitMerkleProof, ZerokitMerkleTree, MIN_PARALLEL_NODES};
-
+use crate::{FromConfigError, ZerokitMerkleTreeError};
```

Comment:
`utills` is an independent library and I think it would be logical to define errors from this library in it as well